test/test_http_cookie.rb

```diff
@@ -1168,4 +1168,11 @@ def test_s_path_match?
     assert_equal true,  HTTP::Cookie.path_match?('/admin', '/admin/')
     assert_equal true,  HTTP::Cookie.path_match?('/admin', '/admin/index')
   end
+
+  def test_to_h_coercion
+    cookie = HTTP::Cookie.new(cookie_values)
+
+    cookie.keys.each { |key| assert_equal key.class, Symbol }
+    assert_equal cookie.keys.map(&:to_s), PERSISTENT_PROPERTIES
+  end
```

Comment:
Can you please add a test for the hash values to this test? Getting the keys right is obviously important but isn't testing the entirety of the returned Hash.